misc/services/acrn_manager/README.rst

```diff
@@ -166,12 +170,12 @@ and sets an RTC timer to wake up the Service VM or bring the Service VM back up
 When ``acrnd`` daemon is restarted, it restores the previously saved timer
 list and launches the User VMs at the right time.
 
-A ``systemd`` service file (``acrnd.service``) is installed by default that will
-start the ``acrnd`` daemon when the Service VM (Linux-based) comes up.
-You can restart/stop acrnd service using ``systemctl``
+A ``systemd`` service file (``acrnd.service``) is installed by default.
+You can enable/restart/stop acrnd service using ``systemctl``.
```

Comment:
Better would be "enable, restart, or stop"

Reply:
Thanks, will update this.